contentcuration/contentcuration/assets/channelList/index.js

```diff
@@ -0,0 +1,42 @@
+import Vue from 'vue';
+import VueRouter from 'vue-router';
+import Vuetify from 'vuetify';
+import ChannelListIndex from './views/ChannelListIndex.vue';
+import store from './store';
+import router from './router';
+
+import 'vuetify/dist/vuetify.min.css';
+import 'shared/styles/main.less';
+
+Vue.use(VueRouter);
+Vue.use(Vuetify);
+
+Vue.filter('kindIcon', function(kind) {
```

Comment:
Is there a technical reason we cannot make this change now? The progress tracking code sometimes throws `multiple Vue.use(Vuex) call` errors because we're instantiating bits of Vue in several different places. Would be nice to get that fixed as part of this!

Reply:
Mostly that we need a root Vue component to do this most effectively, so it is most readily done during the refactor that changes the root of the single page apps to be instantiated by Vue, not by Backbone.

Reply:
I think that given it is just showing a `console.error` and not actually throwing an error, we can probably live with it in the existing code base and not worry about this.

Reply:
It was historic - I would be quite happy for us to collapse Studio into a single page app completely, but because we're not at that stage currently, we probably need to do it in stages, as otherwise we would be waiting on vuefying everything to introduce the vueficiation of the main user facing channel list and channel edit pages.

Reply:
Okay thanks for explanation, yes that makes sense.